packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -19,6 +19,8 @@ enum Artifact {
   genSnapshot,
   /// The flutter tester binary.
   flutterTester,
+  /// A flutter tester binary without raster cache or fractional translation.
+  flutterTesterFractionalTranslation,
```

Comment:
Is the idea that after a full internal migration, we would go back to only having one flutterTester binary?

Reply:
yes, assuming it works 😄